dask/bytes/__init__.py

```diff
@@ -1,17 +1,3 @@
-from distutils.version import LooseVersion
+from fsspec.core import open as open_file, open_files  # noqa
```

Comment:
That's a good point. cc @martindurant  if you're aware of any projects with use these imports directly

Reply:
I am not aware. It was certainly done like this for compatibility reasons - because all those functions used to be defined here. However, `fsspec` is used in many IO places, so it's definitely required. Some imports within dask might still point to this location rather than upstream (which is what this PR is about...).

Reply:
I did that